.gitignore

```diff
@@ -32,3 +32,7 @@ npm-debug.log
 # we ignore priv/static. You may want to comment
 # this depending on your deployment strategy.
 /priv/static/
+
+.iex.exs
```

Comment:
This one was already noted in another review, it doesn't belong with this change :) If you can make the PR only be a minimal change on top of main branch that's the best.

So you could check out main, pull the latest, make your changes for credo. Make a new branch for that, commit and push with those minimal changes. If you run `git diff` before `git add` you can see what you have changed. If you run `git show` you can see what you latest commit added.